tests/desktop/consumer_pages/test_home_page.py

```diff
@@ -52,26 +52,31 @@ def test_that_verifies_categories_menu(self, mozwebqa):
 
     @pytest.mark.sanity
     @pytest.mark.nondestructive
-    def test_opening_every_category_page_from_categories_menu(self, mozwebqa):
+    def test_opening_category_pages_from_categories_menu(self, mozwebqa):
+        """Open the first 3 category pages and check the first 3 apps on those pages."""
 
         home_page = Home(mozwebqa)
         home_page.go_to_homepage()
 
-        for i in range(home_page.category_count):
+        categories = home_page.categories.items
+        for c in range(3):
             home_page.hover_over_categories_menu()
-            category_name = home_page.categories.items[i].name
-            category_page = home_page.categories.items[i].click_category()
+            category = categories[c]
+            category_name = category.name
+            category_page = category.click_category()
             Assert.equal(category_name.title(), category_page.category_title)
             Assert.true(category_page.is_the_current_page)
-            Assert.true(len(category_page.apps) > 0)
+            apps = category_page.apps
+            Assert.true(len(apps) > 0)
```

Comment:
Just to verify, @krupa is OK with doing a less verbose verification, only verifying the apps `count is greater than 0`?